weave/trace_server/interface/base_object_classes/base_object_registry.py

```diff
@@ -1,3 +1,4 @@
+from weave.trace_server.interface.base_object_classes.actions import ActionDefinition
 from weave.trace_server.interface.base_object_classes.base_object_def import BaseObject
```

Comment:
Is this concept new?  Why do we have and register base objects?

Reply:
yes, just landed: https://github.com/wandb/weave/pull/2826